xonsh/xonfig.py

```diff
@@ -740,10 +757,10 @@ def TAGLINES():
     ("{{INTENSE_BLACK}}", "<", "-"),
     "{{GREEN}}xonfig{{NO_COLOR}} tutorial    {{INTENSE_WHITE}}->    Launch the tutorial in "
     "the browser{{NO_COLOR}}",
-    "{{GREEN}}xonfig{{NO_COLOR}} wizard      {{INTENSE_WHITE}}->    Run the configuration "
-    "wizard and claim your shell {{NO_COLOR}}",
-    "{{INTENSE_BLACK}}(Note: Run the Wizard or create a {{RED}}~/.xonshrc{{INTENSE_BLACK}} file "
-    "to suppress the welcome screen)",
+    "{{GREEN}}xonfig{{NO_COLOR}} web         {{INTENSE_WHITE}}->    Run the configuration "
+    "tool in the browser and claim your shell {{NO_COLOR}}",
```

Comment:
Also, you have an extra `i` in claim. I feel like "Clam your Shell" is a good pun.